scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -63,6 +96,19 @@ else
     sudo yum install -y cuda-drivers \
         cuda
 fi
+
+if [[ $AMI_TYPE == "nvidiaopen" ]]; then
+    # remove closed-source nvidia kernel module
+    MODULE_NAME="nvidia"
+    MODULE_VERSION=$(${DKMS} status -m ${MODULE_NAME} | awk '{print $2}' | tr -d ',:')
+    sudo ${DKMS} remove -m "${MODULE_NAME}" -v "${MODULE_VERSION}" --all
+
+    # load open module from tarball
+    MODULE_NAME="nvidia-open"
+    MODULE_ARCHIVE="${DKMS_ARCHIVE_DIR}/${MODULE_NAME}/*.tar.gz"
+    sudo ${DKMS} ldtarball ${MODULE_ARCHIVE}
```

Comment:
and finally install:


# package is installed to nvidia dir (not nvidia-open)
MODULE_NAME="nvidia"
MODULE_VERSION=$(${DKMS} status -m ${MODULE_NAME} | awk '{print $2}' | tr -d ',:')
sudo ${DKMS} install -m "${MODULE_NAME}" -v "${MODULE_VERSION}"